### ISSUE TITLE
SA1115 (ParameterMustFollowComma) incorrectly reported after multiline parameter

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#923]
SA1115 should not be reported for the following statement. However, I am currently seeing it reported 4 times.

```csharp
return Nfa.Choice(
    Nfa.Sequence(
        Nfa.Match(ANTLRParser.TOKEN_REF),
        Nfa.Optional(Nfa.Rule(Bindings.ElementOptions)),
        Nfa.Optional(Nfa.Match(ANTLRParser.ARG_ACTION))),
    Nfa.Match(ANTLRParser.RULE_REF),
    Nfa.Sequence(
        Nfa.Match(ANTLRParser.CHAR_LITERAL),
        Nfa.Optional(Nfa.Rule(Bindings.ElementOptions))),
    Nfa.Sequence(
        Nfa.Match(ANTLRParser.STRING_LITERAL),
        Nfa.Optional(Nfa.Rule(Bindings.ElementOptions))),
    Nfa.Sequence(
        Nfa.Match(ANTLRParser.DOLLAR),
        Nfa.Rule(Bindings.Label)),
    Nfa.Match(ANTLRParser.ACTION));
```

**Comment by pdelvo** [comment]
I think this issue can be closed
